zoo/classic_control/cartpole/envs/cartpole_lightzero_env.py

```diff
@@ -87,6 +94,10 @@ def reset(self) -> Dict[str, np.ndarray]:
 
         action_mask = np.ones(self.action_space.n, 'int8')
         obs = {'observation': obs, 'action_mask': action_mask, 'to_play': -1}
+        if self._enable_chance:
+            chance_value = random.randint(0, 2)
+            obs['observation'][chance_value] *= 2
```

Comment:
Add a comment like this: This is to artificially introduce randomness in order to evaluate the performance of stochastic_muzero on state input.

Reply:
added comment

Reply:
Thanks for your contribution!